exercises/concept/welcome-to-tech-palace/.docs/instructions.md

```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+There is an appliance store called "Tech Palace" nearby. The owner of the store installed a big display that consists of lots of small LED lights. The display can show multiple lines of text and the store owner wants to use it to show a special greetings in case a customer scans their loyalty card at the entrance.
+
+The store owner needs your help with the code that is used to generate the text for the new display.
+
+## 1. Create the welcome message
+
+For the normal customers, the store owner wants to see `Welcome to the Tech Palace, ` following by the name of the customer in capital letters on the display.
```

Comment:
```suggestion
For most customers who scan their loyalty cards, the store owner wants to see `Welcome to the Tech Palace, ` following by the name of the customer in capital letters on the display.
```

"normal" has negative connotations to many people. I also added reference again to the loyalty card as I wondered inititally how the owner knew their names